sdks/python/apache_beam/runners/common.py

```diff
@@ -533,6 +550,7 @@ def counter_for(self, aggregator):
 
 # TODO(robertwb): Replace core.DoFnContext with this.
```

Comment:
@sb2nov - Unrelated to PR. Is this another opportunity for removing backward compatibility code?